templates/CRM/Mailing/Form/Search.tpl

```diff
@@ -63,6 +63,15 @@
         {/if}
     </tr>
 
+    {* language *}
+    {if $form.language}
+      <tr>
+        <td>{$form.language.label} {help id="id-language"}<br />
+            {$form.language.html|crmAddClass:big}
```

Comment:
I think the idea here is to have a consistent UX with all form item at the same size but i see that there is already an exception with the campaign list. So i guess i'm not that much attached to one solution or the other.